bin/wsl-init

```diff
@@ -184,181 +132,83 @@ groups $WSL_USER_GROUP
 # Install Node
 apt-get install -y nodejs
 /usr/bin/npm install -g npm
-#/usr/bin/npm install -g gulp-cli
-#/usr/bin/npm install -g bower
-#/usr/bin/npm install -g grunt-cli
-
-## Install rabbitmq-server and its dependencies
-#apt-get install rabbitmq-server -y --fix-missing
 
 # Install SQLite
 apt-get install -y sqlite3 libsqlite3-dev
 
-# Install MySQL
-echo "mysql-server mysql-server/root_password password secret" | debconf-set-selections
-echo "mysql-server mysql-server/root_password_again password secret" | debconf-set-selections
-apt-get install -y mysql-server
+# Disable Apparmor
+# See https://github.com/laravel/homestead/issues/629#issue-247524528
+service apparmor stop
+update-rc.d -f apparmor remove
+
+# Remove MySQL
+apt-get remove -y --purge mysql-server mysql-client mysql-common
+apt-get autoremove -y
+apt-get autoclean
+
+rm -rf /var/lib/mysql/*
+rm -rf /var/log/mysql
+rm -rf /etc/mysql
 
-# Configure MySQL 8 Remote Access and Native Pluggable Authentication
-cat > /etc/mysql/conf.d/mysqld.cnf << EOF
+# Add Maria PPA
+curl -LsS https://downloads.mariadb.com/MariaDB/mariadb_repo_setup | sudo bash
+
+echo "mariadb-server mysql-server/data-dir select ''" | debconf-set-selections
+echo "mariadb-server mysql-server/root_password password secret" | debconf-set-selections
+echo "mariadb-server mysql-server/root_password_again password secret" | debconf-set-selections
+
+mkdir  /etc/mysql
+touch /etc/mysql/debian.cnf
+
+# Install MariaDB
+apt-get install -y mariadb-server mariadb-client
+
+# Configure Maria Remote Access and ignore db dirs
+sed -i "s/bind-address            = 127.0.0.1/bind-address            = 0.0.0.0/" /etc/mysql/mariadb.conf.d/50-server.cnf
```

Comment:
Isn't this redundant since the entire file is being replaced in the next command (cat)?